src/components/label/_label.scss

```diff
@@ -17,7 +17,7 @@
   }
 
   &--placeholder {
-    font-size: 1rem;
+    font-size: 0.95rem;
```

Comment:
I don't think we need to do this as we don't support below 325px

Reply:
I spoke to Jon about this because I wasn't sure what the minimum we support was, he said its usually 375px because that was the smallest smartphone but the Galaxy Fold is now only 280px

Reply:
If we support 280px then loads of things will break as our `xxs` breakpoint is 300px currently. I don't think we should support this size as yet as its not a mainstream thing.

Reply:
Ok fair enough, probably best have a chat with Jon, if you guys decide to not support anything below that then let me know and Ill remove this change from the PR

Reply:
OK, let's consider/deal with smaller viewport support in a separate issue, because like Matt says, we'd need to adjust the smallest breakpoints and change much more to fully support it.
We'd probably need to make the assessment based services' analytics.